Sources/WalletConnectKMS/Crypto/CryptoKitWrapper/AgreementCryptoKit.swift

```diff
@@ -34,6 +37,18 @@ public struct AgreementPublicKey: GenericPasswordConvertible, Equatable {
         try self.init(rawRepresentation: data)
     }
 
+    public init(base64url: String) throws {
+        var base64 = base64url
```

Comment:
we can reuse it here and in JWTEncoder. With Base64Encoder object or something like this

Reply:
or maybe it's better to just add a Data object extension?